cobalt/base/startup_timer.cc

```diff
@@ -14,7 +14,7 @@
 
 #include "cobalt/base/startup_timer.h"
 
-#include "starboard/once.h"
+#include "starboard/common/once.h"
```

Comment:
If you're going to unconditionally include starboard/common/once.h doesn't that need to include starboard/once.h?

Reply:
No, the starboard/once.h is gone. None of the code needs it.